util/cron/test-gpu-ex-cuda-12.bash

```diff
@@ -12,6 +12,7 @@ export CHPL_LLVM=bundled  # CUDA 12 is only supported with bundled LLVM
 export CHPL_COMM=none
 export CHPL_LOCALE_MODEL=gpu
 export CHPL_LAUNCHER_PARTITION=allgriz
+export CHPL_TEST_GPU=true
```

Comment:
This might have been something you intended to remove?

I think this setting comes from one of the common scripts.

Reply:
Yeah, thanks!